src/editable-heading/editable-heading.stories.tsx

```diff
@@ -106,7 +106,7 @@ export const basic = () => {
       </ExampleEditableHeading>
       {lorem}
 
-      <ExampleEditableHeading multiline maxInputHeight={150}>
+      <ExampleEditableHeading multiline>
```

Comment:
I think it's a good idea to add the `maxInputRows` here or to make a new story in which that attribute will be passed.

Reply:
done

Reply:
Thanks!